pmm-app/tests/pages/pmmSettingsPage.js

```diff
@@ -116,7 +116,23 @@ module.exports = {
     tooltipSelector: '.popper__background',
     validationMessage: 'span.error-message',
     tabsSection: '[data-qa="settings-tabs"]',
-    tabContent: '[data-qa="settings-tab-content"]'
+    tabContent: '[data-qa="settings-tab-content"]',
```

Comment:
Could you please sort these locators, the current fields elements are sorted, please follow the same pattern.